### PR TITLE
ongoing pre-beta tasks

### PR DESCRIPTION
**Pull request by akinwale**

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
         "react-native-fast-image": "^5.0.3",
         "react-native-fetch-blob": "^0.10.8",
         "react-native-image-zoom-viewer": "^2.2.5",
-        "react-native-phone-input": "^0.2.1",
+        "react-native-phone-input": "lbryio/react-native-phone-input",
```
Comment (skhameneh) [inline app/package.json]: Can you make an upstream request?
We have a lot of repos and it would be great to get maintained upstream fixes

Comment (akinwale) [reply app/package.json]: Upstream was last updated 6 months ago and there's been no response from the repo owner. I was thinking we could probably have a separate github org for forked repos.

Comment (kauffj) [reply app/package.json]: Alternatively, and so long as it's not against the license to do so, just check the code directly into our repo.

**Comment by akinwale** [comment]
@skhameneh This is ready to merge. Please let me know if you have any more comments.

**Comment by akinwale** [comment]
I'll merge this one now. Please let me know if you eventually find any issues.

**Pull request merged**